models/Loans.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require('mongoose');
+
+let loanSchema = new mongoose.Schema({ 
+    users_id: {
+        type: mongoose.Schema.Types.ObjectId,
```

Comment:
See this
https://alexanderzeitler.com/articles/mongoose-referencing-schema-in-properties-and-arrays/